### PR TITLE
Footer template

### PR DESCRIPTION
**Pull request by ebullient**
Draft footer template. Please revise (do we want a logo or not?)

### CODE REVIEW

File: templates/website-footer.md [hunk 1]
```diff
@@ -5,8 +5,25 @@ status: draft
...
 
-## TBD: Example text
+## HTML
 
-> Copyright © [Commonhaus Foundation](https://website) and [project name] contributors. All rights reserved. The [Commonhaus Foundation](https://website) has registered trademarks and uses trademarks. For a comprehensive list of Commonhaus Foundation trademarks, consult our [Trademark Policy](https://trademark-policy.website/) and [Trademark List](https://trademark-list.website/). Any trademarks or logos not listed on the [Commonhaus Foundation trademarks list](https://trademark-list.website) are trademarks™ or registered® trademarks owned by their respective holders. Their use does not suggest any affiliation with or endorsement by them.
+```html
+<p>
+  Copyright © <a href="https://www.commonhaus.org">Commonhaus Foundation</a>. All rights reserved. For details on our trademarks, please visit our <a href="/policies/trademark-policy/">Trademark Policy</a> and <a href="/trademarks/">Trademark List</a>. Trademarks of third parties are owned by their respective holders and their mention here does not suggest any endorsement or association.<br />
```
Comment (kenfinnigan) [inline templates/website-footer.md:12]: Should the trademark policy and list links be the full href like they are below?

Comment (ebullient) [reply templates/website-footer.md:12]: oh. oops. yes.

File: templates/website-footer.md [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
+```
+<!-- markdownlint-disable MD033 -->
+<p><img src="https://raw.githubusercontent.com/commonhaus/artwork/main/foundation/brand/svg/CF_logo_horizontal_single_default.svg" height="30" alt=""/><br />Copyright © <a href="https://www.commonhaus.org">Commonhaus Foundation</a>. All rights reserved. For details on our trademarks, please visit our <a href="https://www.commonhaus.org/policies/trademark-policy/">Trademark Policy</a> and <a href="https://www.commonhaus.org/trademarks/">Trademark List</a>. Trademarks of third parties are owned by their respective holders and their mention here does not suggest any endorsement or association.</p>
+
+## Markdown
```
Comment (aalmiray) [inline templates/website-footer.md:15]: An Asciidoc option would be great as well

Comment (ebullient) [reply templates/website-footer.md:15]: Done.

**Review by aalmiray — APPROVED** [review]
LGTM now that we have 3 formatted options

**Pull request merged**